src/java.base/share/classes/com/sun/crypto/provider/GCTR.java

```diff
@@ -139,4 +139,102 @@ int doFinal(byte[] in, int inOfs, int inLen, byte[] out,
         }
         return inLen;
     }
-}
+
+    // Maximum buffer size rotating ByteBuffer->byte[] intrinsic copy
+    final int MAX_LEN = 1024;
```

Comment:
This filed could be private. I would like to declare class fields in the beginning of a class, for easy eyes-searching. Maybe, it is nice to use a multiple of the block size (for example 64 * AES_BLOCK_SIZE), just in case someone else update it to a weird value later.